trickle/trickle_server.go

```diff
@@ -438,7 +438,15 @@ func (s *Stream) getForRead(idx int) (*Segment, int, bool) {
 		return seg != nil && seg.idx == i
 	}
 	if idx == -1 {
```

Comment:
Did the `-1` behavior change with that change from `s.lastWrite` to `s.nextWrite` (IIRC) or has it always been like this (picking the next, unstarted seq)?

Reply:
It's always been like this with client preconnect behavior, I tried to explain in https://github.com/livepeer/go-livepeer/pull/3548#issuecomment-2852141912 but could have done a poor job of it, LMK if anything is unclear there